### DISCUSSION TITLE
Should REFRESH_TOKEN_COOKIE_SECURE in the example.env be "false" or `false`?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13353] [Archive]
REFRESH_TOKEN_COOKIE_SECURE is set to "false" in the [`example.env`](https://raw.githubusercontent.com/directus/directus/f26d862a35c587cd3fd6f15d8e9b6469d16489c7/api/example.env) - should it be `false` or does it not matter?

**Comment by azrikahar** [comment]
they are actually both treated as string! And this is true for environment variables in general, not just limited to Directus or Node (perhaps some other languages may do conversion out of the box? not that I'm aware of). 

However, Directus does ultimately convert them to actual boolean over here:

https://github.com/directus/directus/blob/726af0c57cf4e3a94c61d828db0dbd811851c4c9/api/src/env.ts#L275-L283

**Reply by sxsws** [reply]
Sure, just wanted to query it just in case!
